astrolabe/runner.py

```diff
@@ -214,6 +256,11 @@ def run(self, persist_cluster=False, startup_time=1):
             # is only visible for failed tests.
 
         LOGGER.info("Workload Statistics: {}".format(stats))
+        
+        #LOGGER.info("Waiting 5 minutes for Atlas logs to become available")
+        #sleep(5*60)
+        
```

Comment:
This looks like debug code that should be removed.
```suggestion
```

Reply:
Actually this was needed prior to the current method of log retrieval, but currently this code has no purpose.